testnets/elystestnet/assetlist.json

```diff
@@ -74,6 +66,97 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/elystestnet/images/eden.png"
         }
       ]
+    },
+    {
+      "description": "USDC from Noble",
+      "denom_units": [
+        {
+          "denom": "ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65",
+          "exponent": 0
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65",
+      "name": "Noble USD Coin",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble",
+            "base_denom": "uusdc",
+            "channel_id": "channel-19"
+          },
+          "chain": {
+            "channel_id": "channel-12",
+            "path": "transfer/channel-12/uusdc"
```

Comment:
Neither of the IBC connections are registered. I assume you meant to actually put nolustestnet and cosmoshubtestnet ?

Reply:
Just updated the names. Is it ok now? These are Noble Testnet and Cosmos Testnet IBC connections w/ Elys

Reply:
Thanks for checking

Reply:
Thanks for correcting the name. Although neither of these IBC connections are registered here: https://github.com/cosmos/chain-registry/tree/master/testnets/_IBC

Reply:
Updated. thanks